scripts/tasks/flow-ci.js

```diff
@@ -14,12 +14,22 @@ process.on('unhandledRejection', err => {
 const runFlow = require('../flow/runFlow');
 const inlinedHostConfigs = require('../shared/inlinedHostConfigs');
 
+// Parallelize tests across multiple tasks.
+const nodeTotal = process.env.CIRCLE_NODE_TOTAL
+  ? parseInt(process.env.CIRCLE_NODE_TOTAL, 10)
+  : 1;
+const nodeIndex = process.env.CIRCLE_NODE_TOTAL
```

Comment:
Did you mean to check NODE_TOTAL again here or should it be:

```suggestion
const nodeIndex = process.env.CIRCLE_NODE_INDEX
```